### DISCUSSION TITLE
Questions about EMC2

### DISCUSSION THREAD

**Discussion by niekstevenson** [ampl-psych/EMC2#76] [General]
Feel free to ask your questions!

**Comment by priyamvadamodak** [comment]
The help for DDM shows that the range for sv as [0, Inf], but the default value is shown as log(0) which is -Inf. Hopefully, it is just a typing error. Same for st0.

**Reply by frankhezemans** [reply]
That is not a typo: It reflects the distinction between the parameter's _sampled scale_ and _natural scale_. To improve sampling performance, all parameters are initially sampled on the real line (-Inf, Inf). Then, to evaluate the model's likelihood, parameters are transformed to their natural scale using suitable monotonic functions. For the DDM, the sv parameter is transformed from the real line (-Inf, Inf) to the strictly positive scale [0, Inf) using the exponential function. Hence, the default value of -Inf on the sampled scale is transformed to 0 on the natural scale.

**Comment by DavideGheza** [comment]
FYI, to install on your local R, use (temporarily) the dev version
remotes::install_github("ampl-psych/EMC2@dev",dependencies=TRUE) # dev version of EMC2

**Comment by j-duff** [comment]
Hey EMC2 team - thanks for making and maintaining this tool, I'm using it for a couple of hierarchical DDM models and it's working very nicely. 

I just implemented a group design for the first time, to allow for effects of which of two sites the participant was run at (sum-coded), on v, a, or t0. Fit very nicely (much better than without the group design), but I noticed my summary() calls, plot_pars(), etc. don't show any posteriors for the effects of that Site factor. I'd like to see and report them---Is there any undocumented parameter I can pass to get them included in the output there, or any tips for how to extract some means and CRIs for those effects manually from the fitted EMC object?
